pymc3/tests/test_parallel_sampling.py

```diff
@@ -87,15 +85,13 @@ def test_remote_pipe_closed():
             pm.sample(step=step, mp_ctx="spawn", tune=2, draws=2, cores=2, chains=2)
 
 
-@pytest.mark.xfail(
-    reason="Possibly the same issue described in https://github.com/pymc-devs/pymc3/pull/4701"
-)
+@pytest.mark.xfail(reason="Unclear")
```

Comment:
Let's open a separate issue to investigate this. The other two tests seem to pass after the changes I added

Reply:
Yeah, those fails are unrelated to the fix in this PR.